src/pages/Profile.tsx

```diff
@@ -46,16 +50,38 @@ const Profile = () => {
 
   const actions = [
     {
-      key: 'settings',
-      label: stringGetter({ key: STRING_KEYS.SETTINGS }),
-      icon: { iconName: IconName.Gear },
-      href: AppRoute.Settings,
+      key: 'deposit',
+      label: stringGetter({ key: STRING_KEYS.DEPOSIT }),
+      icon: { iconName: IconName.Deposit },
+      onClick: () => {
+        abacusStateManager.setTransferValue({
+          field: TransferInputField.type,
+          value: TransferType.deposit.rawValue,
+        });
+        dispatch(openDialog({ type: DialogTypes.ManageFunds }));
+      },
     },
-    isConnected && {
-      key: 'transfers',
-      label: stringGetter({ key: STRING_KEYS.MANAGE_FUNDS }),
-      icon: { iconName: IconName.Transfer },
+    {
+      key: 'withdraw',
+      label: stringGetter({ key: STRING_KEYS.WITHDRAW }),
+      icon: { iconName: IconName.Withdraw },
       onClick: () => {
+        abacusStateManager.setTransferValue({
+          field: TransferInputField.type,
+          value: TransferType.withdrawal.rawValue,
+        });
+        dispatch(openDialog({ type: DialogTypes.ManageFunds }));
+      },
+    },
+    {
+      key: 'transfer',
+      label: stringGetter({ key: STRING_KEYS.TRANSFER }),
+      icon: { iconName: IconName.Send },
+      onClick: () => {
+        abacusStateManager.setTransferValue({
+          field: TransferInputField.type,
+          value: TransferType.transferOut.rawValue,
+        });
         dispatch(openDialog({ type: DialogTypes.ManageFunds }));
```

Comment:
Instead of adding `setTransferValue` to the onClick, can we move this to the `ManageFunds` dialog so that we don't have to call the abacus method prior to opening dialogs?